lib/boot-record.coffee

```diff
@@ -60,7 +60,7 @@ exports.parse = (mbrBuffer) ->
 #	bootRecord.getExtended('path/to/rpi.img', 2048).then (ebr) ->
 #		console.log(ebr)
 ###
-exports.getExtended = (image, position) ->
+exports.getExtended = getExtended = (image, position) ->
```

Comment:
Can we remove the local declaration in these functions? They are not being used in this file.